packages/eslint-plugin-angular/package.json

```diff
@@ -26,13 +26,14 @@
         "build": "tsc",
         "test": "echo \"Error: run tests from root\" && exit 1"
     },
+    "dependencies": {
+        "@angular-eslint/eslint-plugin": "13.5.0",
+        "@angular-eslint/eslint-plugin-template": "13.5.0",
+        "@angular-eslint/template-parser": "13.5.0"
+    },
     "peerDependencies": {
-        "@angular-eslint/eslint-plugin": ">=13.0.0",
-        "@angular-eslint/eslint-plugin-template": ">=13.0.0",
-        "@angular-eslint/template-parser": ">=13.0.0",
         "@cloudflight/eslint-plugin-typescript": ">=0.1.0",
-        "@typescript-eslint/parser": ">=5.0.0 <6.0.0",
-        "eslint": ">=8.0.0 <9.0.0"
+        "eslint": "8.22.0"
```

Comment:
why would you explicitly restrict it to one exact version?

Reply:
To prevent any potential breakage.

Reply:
Especially between typescript-eslint and eslin itself.

Reply:
yes but that forces a project to use exactly this version, which might be an issue for them.

In my opinion `peerDependencies` should always be as lenient as possible and avoid depending on a fixed version, then that could be a normal dependency from the start.

Reply:
Fixed.

Reply:
eslint-plugin-typescript still has the exact version

Reply:
Fixed.